ui/src/components/Comment/components/Form/index.tsx

```diff
@@ -83,6 +83,7 @@ const Index = ({
               size="sm"
               value={type === 'edit' ? parseEditMentionUser(value) : value}
               onChange={handleChange}
+              isInvalid={validationErrorMsg !== ''}
```

Comment:
`ui/src/components/Comment/components/Reply/index.tsx`, this file also needs to be adjusted!

Reply:
Done. Please verify